safe_transaction_service/safe_messages/tests/factories.py

```diff
@@ -1,22 +1,23 @@
 import factory
 from eth_account import Account
-from eth_account.messages import defunct_hash_message, encode_defunct
 from factory.django import DjangoModelFactory
 
 from ..models import SafeMessage, SafeMessageConfirmation
+from ..utils import get_safe_message_hash_for_message
 
 
 class SafeMessageFactory(DjangoModelFactory):
     class Meta:
         model = SafeMessage
 
     safe = factory.LazyFunction(lambda: Account.create().address)
-    message_hash = factory.LazyAttribute(
-        lambda o: defunct_hash_message(text=o.message).hex()
-    )
     message = factory.Sequence(lambda n: f"message-{n}")
     proposed_by = factory.LazyFunction(lambda: Account.create().address)
-    description = factory.Faker("sentence", nb_words=5)
+    safe_app_id = factory.Sequence(lambda n: n)
```

Comment:
Yeah, I used `Sequence` as it's more predictable during testing, but random integer would work for me. Do you think random is better in this case?

Reply:
> Yeah, I used `Sequence` as it's more predictable during testing, but random integer would work for me. Do you think random is better in this case?

I don't think it's better. When I saw `Sequence` it seemed that we had to have this assumption (different ids generated on the factory). But I agree that this might make things easier to test 👍